lib/game.rb

```diff
@@ -0,0 +1,74 @@
+class Game
+
+  WIN_COMBINATIONS = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [2,4,6]
+  ]
+
+  attr_accessor :player_1, :player_2, :board
+
+  def initialize(player_1=Players::Human.new("X"),
+                 player_2=Players::Human.new("O"),
+                 board=Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    self.board.turn_count.even? ? self.player_1 : self.player_2
+  end
+
+  def won?
+    cells = self.board.cells
+    WIN_COMBINATIONS.each do |combo|
+      if self.board.taken?((combo[0] + 1).to_s) &&\
+         cells[combo[0]] == cells[combo[1]] &&\
+         cells[combo[0]] == cells[combo[2]]
+        return combo
+      end
+    end
+    nil
+  end
+
+  def over?
+    self.won? || self.draw?
+  end
+
+  def draw?
+    !self.won? && self.board.full?
+  end
+
+  def winner
+    self.won? && self.board.cells[self.won?[0]]
```

Comment:
Instead of calling `Game#won?` twice, consider using an if statement assignment. For example:

```ruby
if won = won?
  board.cells[won.first]
end
```

If `won?` returns true, then and only then will the local variable `won` be assigned, otherwise, the method will just abort with a false return value. But if `won` is assigned, we can use that variable instead of calling `won?` again within the if statement.